webapp/locales/de.json

```diff
@@ -269,7 +269,8 @@
         "comment": {
             "submit": "Kommentiere",
             "submitted": "Kommentar Gesendet",
-            "updated": "Änderungen gespeichert"
+            "updated": "Änderungen gespeichert",
+            "answered": "answered"
```

Comment:
Please translate.
I would name this `"answer": „Antworten“`.

Reply:
ok